scripts/run-system-tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
How do you plan to be incorporate this scipt to cloudify-system-tests? 
Are you going to fetch it from cloudify-ui-common or will it be added to cloudify-stage and cloudify-blueprint-composer?

Reply:
Since stage and composer both depend on cloudify-ui-common I was just planning to add a npm script `    "e2e:ci": "node_modules/cloudify-ui-common/scripts/run-system-tests.sh",` to both and run `npm run e2e:ci` in related python test suites